src/codewhisperer/vue/backend.ts

```diff
@@ -0,0 +1,147 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import * as nls from 'vscode-nls'
+const localize = nls.loadMessageBundle()
+import * as os from 'os'
+import * as vscode from 'vscode'
+import * as path from 'path'
+import { VueWebview } from '../../webviews/main'
+import { isCloud9 } from '../../shared/extensionUtilities'
+import globals from '../../shared/extensionGlobals'
+import { telemetry, CodewhispererLanguage, CodewhispererGettingStartedTask } from '../../shared/telemetry/telemetry'
+import { FileSystemCommon } from '../../srcShared/fs'
+export type OSType = 'Mac' | 'RestOfOS'
+export class CodeWhispererWebview extends VueWebview {
+    public readonly id = 'CodeWhispererWebview'
+    public readonly source = 'src/codewhisperer/vue/index.js'
+
+    public constructor(private readonly start: string) {
+        super()
+    }
+    //This function is called when the extension is activated to check whether is it the first time the user is using the extension or not
+    public async showAtStartUp(): Promise<string | void> {
+        return this.start
+    }
+
+    private isFileSaved: boolean = false
+    private getLocalFilePath(fileName: string): string {
+        //This will store the files in the global storage path of VSCode
+        return path.join(globals.context.globalStorageUri.fsPath, fileName)
+    }
+
+    // This function opens Python/JavaScript/C# file in the editor.
```

Comment:
comment is incorrect to not reflecting all 5 languages.

Reply:
Updated to all the 5 languages  [ TypeScript/JavaScript/Python/Java/C# ]